paddle/framework/operator.h

```diff
@@ -281,51 +281,47 @@ class OpKernel {
 
 class OperatorWithKernel : public OperatorBase {
  public:
-  struct OpKernelKey {
-    platform::Place place_;
-
-    OpKernelKey() = default;
-    explicit OpKernelKey(const platform::DeviceContext& dev_ctx) {
-      place_ = dev_ctx.GetPlace();
-    }
-
+  struct OpKernelKey : public platform::Place {
```

Comment:
OpKernelKey will not only depend on `Place`, It will also consider `data_type` in the future, such as `float32/float16`, so I think we should not make it the subclass of Place